CHANGES.md

```diff
@@ -3,6 +3,11 @@
 
 ## CKEditor 4.11.4
 
+Fixed Issues:
+
+* [#2816](https://github.com/ckeditor/ckeditor-dev/issues/2816): Fixed: [Enhanced Image](https://ckeditor.com/cke4/addon/image2) resize handler is not created in [Read-Only Mode](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_readonly.html) editor.
+* [#2874](https://github.com/ckeditor/ckeditor-dev/issues/2874): Fixed: [Enhanced Image](https://ckeditor.com/cke4/addon/image2) resize handler is visible after entering [Read-Only Mode](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_readonly.html) using console command.
```

Comment:
A small suggestion - don't try to predict the usage of the API if the issue is not based on some environment variables (it's something you already tried with `manually setting Read-Only mode`). Will it work correctly if you write in your app source code `editor.setReadOnly( true )` instead of using it in the console?

Rephrase this sentence into 

> [#2874](https://github.com/ckeditor/ckeditor-dev/issues/2874): Fixed: [Enhanced Image](https://ckeditor.com/cke4/addon/image2) resize handler is visible after entering [Read-Only Mode](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_readonly.html).